### PR TITLE
add trust-graph-js project

### PR DESCRIPTION
**Pull request by DieMyst**

### CODE REVIEW

File: js/src/trust_graph.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        let certsStr = [];
+        for (let cert of certs) {
+            certsStr.push(await certificateToString(cert));
+        }
+
+        /*let response = await this.client.callPeer("add_certificates", {
+            certificates: certsStr,
+            peer_id: peerId
+        });*/
+        let response: any = {};
```
Comment (folex) [inline js/src/trust_graph.ts:38]: is that OK?

File: js/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+};
```
Comment (folex) [inline js/jest.config.js:4]: newline

File: js/.npmignore [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+node_modules
+types
+
+src/
+
+tsconfig.json
+webpack.config.js
+
+bundle
+pkg
```
Comment (folex) [inline js/.npmignore:12]: newline

**Review by folex — COMMENTED** [review]
is it OK that package-lock.json is 7k lines?

**Comment by DieMyst** [comment]
> is it OK that package-lock.json is 7k lines?

one libp2p dep and BOOM

**Review by folex — APPROVED** [review]

**Pull request merged**